portfolio/src/main/webapp/html/components/map.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Geolocation</title>
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no">
+    <meta charset="utf-8">
+    <style>
+      /* Always set the map height explicitly to define the size of the div
+       * element that contains the map. */
+      #map {
+        height: 100%;
+      }
+      /* Optional: Makes the sample page fill the window. */
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="map"></div>
+    <script>
```

Comment:
Can you move this to a separate JS file?

Reply:
when I tried it wouldn't work? maybe I was doing it wrong or putting it in the wrong place but it wouldn't show up
all the google maps API documentation has it all on one page so I don't have a reference as to how to do it right